api/src/services/programsService.ts

```diff
@@ -384,3 +384,24 @@ export const setParticipantActivityCompletion = async (
     completed: participantActivity.completed === 1,
   };
 };
+
+//  * Get the completion status (either true or false) of all activities of type assignment in a given program.
```

Comment:
Your function header comments are not to JSDoc/TSDoc standard. Please adjust these to be conformant in the same way that the other function headers in this file are.